core/common/src/TextureProps.ts

```diff
@@ -35,3 +35,15 @@ export interface TextureLoadProps {
   /** Maximum texture size supported by the client. If specified, the texture will be downsampled so both of its dimensions adhere to this size. */
   maxTextureSize?: number;
 }
+
+/** Properties returned with a texture.
+ * @public
+ */
+export interface TextureData {

```

Comment:
Wouldn't this confuse it with the Image class?  I had called it data because it was returning the raw data.  Should we change the function name instead?

Reply:
Sure.

Reply:
Changed function to queryTextureData